ci/docker/install/ubuntu_aarch64_publish.sh

```diff
@@ -63,7 +63,7 @@ make -j$(nproc)
 sudo make install-strip
 cd ..
 rm -rf gcc-8.5.0.tar.xz
-export export PATH=/usr/local/gcc-8.5.0/bin:$PATH
+export PATH=/usr/local/gcc-8.5.0/bin:$PATH
 export LD_LIBRARY_PATH=/usr/local/gcc-8.5.0/lib64:$LD_LIBRARY_PATH
```

Comment:
hey @leezu , I can see `gfortran-8.5` installed in `/usr/local/gcc-8.5.0/bin/` and `libgfortran` binaries in `/usr/local/gcc-8.5.0/lib64/` as part of building gcc-8.5 from source
```
ls /usr/local/gcc-8.5.0/lib64/libgfortran*
/usr/local/gcc-8.5.0/lib64/libgfortran.a   /usr/local/gcc-8.5.0/lib64/libgfortran.so    /usr/local/gcc-8.5.0/lib64/libgfortran.so.5.0.0
/usr/local/gcc-8.5.0/lib64/libgfortran.la  /usr/local/gcc-8.5.0/lib64/libgfortran.so.5  /usr/local/gcc-8.5.0/lib64/libgfortran.spec
```
Still I am getting an error while building openblas that
`error while loading shared libraries: libgfortran.so.5: cannot open shared object file: No such file or directory`
https://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/restricted-mxnet-cd%2Fsrivrohi-release-job-v1.x/detail/srivrohi-release-job-v1.x/120/pipeline/
Any suggestions?

Reply:
you will need that path to be in the LD_LIBRARY_PATH at runtime